src/Opts.hs

```diff
@@ -65,6 +72,14 @@ parser = Options
         <> value 8080
         )
       )
+  <*> ( BlockExplorerPrefix <$> strOption
+        (  long "block-explorer-prefix"
+        <> metavar "URL"
+        <> help "Prefix URL for exploring Cardano blockchain addresses, transactions, etc."
+        <> showDefault
+        <> value "https://preprod.cardanoscan.io/transaction"
+        )
+      )
```

Comment:
Probably even the `https://` and the last `/`, cause otherwise you have to know that you have to write it and it will crash if you don't

Reply:
Agree about the `http://`. I was concerned about the `transactions` part because we probablyi can't expect a uniform shape from one explorer to the next. And that kind of means this single arg is not nearly enough to make this customizable, unfortunately.
But I can make these changes for and we'll see what happens when adding a diff explorer.